src/Uno.UI/Controls/CommandBarNavigationItemRenderer.iOS.cs

```diff
@@ -217,11 +221,17 @@ protected override Size MeasureOverride(Size availableSize)
 						&& !_childSize.Height.IsNaN()
 						&& _childSize.Height != 0
 						&& _childSize.Width != 0
-						&& (Frame.Width != _childSize.Width
-						|| Frame.Height != _childSize.Height))
+						&& ( Frame.Width != _childSize.Width && _childSize.Width < Frame.Width
```

Comment:
Use explicit precedence with parenthesis.

Reply:
I had it and Rider started to complain about them. I will re-add it and will tell Rider to shut up! :D